### ISSUE TITLE
DDL failure for partial indexes

### ISSUE THREAD

**Issue by stevenwinfield** [OmniDB/OmniDB#791]
OmniDB 2.12.0 (windows)
Postgres 11.1 (linux)

```sql
CREATE TABLE test (id integer);
CREATE INDEX idx_test ON test (id);
CREATE INDEX idx_test_partial ON test (id) WHERE id = 1;
```

`idx_test` is displayed properly, but expanding the object tree and selecting `idx_test_partial` pops up an error message saying:

> more than one row returned by a subquery used as an expression

along with, confusingly, a password prompt and OK/Cancel buttons.

**Comment by wind39** [comment]
Hi @stevenwinfield ,

Thanks for reporting. This is already fixed in dev branch, we are going to publish the fix on next release, 2.13.0.

Regards,

William

**Comment by stevenwinfield** [comment]
Great, thanks!
